src/main/java/chess/dao/GameDao.java

```diff
@@ -1,15 +1,25 @@
 package chess.dao;
 
+import chess.controller.dto.response.GameIdentifiers;
 import chess.domain.GameState;
+import java.util.List;
 import java.util.Optional;
 
 public interface GameDao {
 
-    void save(long id);
+    void save(Long id, String name, String password, String salt);
 
-    Optional<GameState> load(long id);
+    List<GameIdentifiers> findAllGames();
```

Comment:
```suggestion
    List<GameIdentifiers> findAll();
```

GameDao인데 game을 또 붙여줄 필요가 있을까요?